index.md

```diff
@@ -75,103 +62,120 @@ Bitte beachtet die folgenden Schritte:
 
 - Die Software vorher [installieren](/install)
   - Bei Problemen können wir gerne per [Mail](about.html) oder beim betreuten Installieren helfen.
-<!--    - Der Workshop wird digital stattfinden, dazu werden wir [Zoom](https://tu-dortmund.zoom.us/){:target="_blank"} (und
-      [Discord](https://discord.com/new/download){:target="_blank"}) verwenden. -->
   - Einen Laptop mitbringen, eine begrenzte Anzahl Laptops kann auch von uns gestellt werden.
   - Verlängerungskabel/Mehrfachstecker mitbringen (falls möglich)
 
 
-## Ablaufplan 2022
+## Ablaufplan 2023
 
+<!--
 An allen Tagen sind wir im Gebäude Bio- und Chemieingenieurwesen ([Lageplan Nr. 6](https://www.tu-dortmund.de/storages/tu_website/Referat_1/News/2019/2019_10_Oktober/191010_TU_Lageplan_A4_WEB_2019.pdf)).
 Der Eingang befindet sich gegenüber vom Haupteingang des Hörsaalgebäudes II.
+-->
 
 <table class="table table-hover">
 <thead>
   <tr>
   <th>Tag</th>
+  <th>Datum</th>
   <th>Zeit</th>
   <th>Raum</th>
```

Comment:
Normally I do that at the start of the summer semester. As exams have priorities.

Reply:
But last time there were these many shifts between rooms, which were a bit anoying, right? 
If it would be possible to prevent that, this would be nice.

If the exams have to move to the other room or we, I don't see why that matters.

Reply:
I will request a room next week, and post the response.

Reply:
We can have ZE HS 01 on every day.

Reply:
Nice!